ModelicaTestConversion4.mo

```diff
@@ -725,35 +725,118 @@ Conversion test for <a href=\"https://github.com/modelica/ModelicaStandardLibrar
 
   package Magnetic
     extends Modelica.Icons.ExamplesPackage;
-    package FundamentalWave
+    package FluxTubes
       extends Modelica.Icons.ExamplesPackage;
       package Issue496 "Conversion test for #194"
         extends Modelica.Icons.ExamplesPackage;
-        partial model PartialTwoPort "Two magnetic ports for graphical modeling"
+        partial model PartialTwoPort
           extends Modelica.Magnetic.FundamentalWave.Interfaces.PartialTwoPort;
           annotation (Documentation(info="<html>
 <p>
 Conversion test for <a href=\"https://github.com/modelica/ModelicaStandardLibrary/issues/496\">#496</a>.
 </p>
-</html>"));
+</html>"),         experiment(Tolerance=1e-06, StopTime = 1));
```

Comment:
No need for this experiment annotation here and further below (if this is a partial model only).